modules/thermal_hydraulics/test/tests/problems/brayton_cycle/gold/closed_brayton_cycle.csv

```diff
@@ -1,32 +1,32 @@
+2.2,0.79,1,-0.6963592587151,0.0069568887293981,-17829.425629785,-2.5368741497079,-0.025183695865122,85159.906092306,-0.15505017753424,-0.18203566708244,0,0,145544.62608732,145603.59639736,145546.96870054,145602.39484779,1.0000160954979,1.0000082522652,100.73478346049,0.843,1,-0.52078577395459,0.0096350104128714,100.73478346049
+2.3,0.79,1,-0.66237338771307,0.006495514970409,-21031.65180729,-2.5377752643489,-0.025188168176491,88325.542005518,-0.14120954963806,-0.18414508327201,0,0,147997.86220903,148057.76973791,148002.00265159,148053.64179633,1.0000279763673,1.0000278813917,100.75267270597,0.843,1,-0.51834562000103,0.0096238775216213,100.75267270597
+2.4,0.79,1,-0.63673368147997,0.0061037018606355,-25448.415031921,-2.5380545209558,-0.025189553990473,91248.20263746,-0.13218272654983,-0.18615094856145,0,0,150415.68313012,150480.94724232,150419.31482955,150476.9538313,1.00002414442,1.0000265383563,100.75821596189,0.843,1,-0.51543232546685,0.0096162399976264,100.75821596189
+2.5,0.79,1,-0.62144125065932,0.0058049624342694,-31295.64220591,-2.5381264588077,-0.025189910970504,94161.28386009,-0.12791314481559,-0.18844993117041,0,0,152748.09548395,152825.08580204,152751.40848795,152821.59118475,1.000021689331,1.0000228673008,100.75964388202,0.843,1,-0.51341149491003,0.0096136957021312,100.75964388202
+2.6,0.79,1,-0.61910548433148,0.0056050516196636,-38871.957107161,-2.5381292871968,-0.025189925005827,97182.317133102,-0.12815857688225,-0.19055620483725,0,0,154942.14149746,155032.84207901,154945.93742194,155028.97672406,1.0000244989803,1.0000249331127,100.75970002331,0.843,1,-0.51119854024368,0.0096170046471615,100.75970002331
+2.7,0.79,1,-0.6299675624877,0.0054927769659023,-48466.07852325,-2.5381090790405,-0.025189824726665,100352.57277976,-0.13217142069536,-0.19220314069304,0,0,156932.7622624,157039.8699799,156937.44282842,157035.61445849,1.0000298252956,1.0000270990847,100.75929890666,0.843,1,-0.50832887008028,0.0096263818861559,100.75929890666
+2.8,0.79,1,-0.65242997905499,0.0054461335021972,-60203.333588319,-2.5380811922856,-0.025189686343251,103703.59791786,-0.13899481065159,-0.19332795306449,0,0,158650.29735246,158777.21797293,158656.14899829,158772.84211269,1.000036883926,1.0000275605083,100.75874537301,0.843,1,-0.50493471022438,0.0096416629785171,100.75874537301
+2.9,0.79,1,-0.68397651360453,0.0054405807824613,-73894.15728764,-2.5380507462677,-0.02518953525905,107220.25934526,-0.14768423707931,-0.19383029938644,0,0,160029.31404374,160177.16128108,160036.48469731,160172.76202372,1.0000448083754,1.0000274657021,100.7581410362,0.843,1,-0.50103139119596,0.0096624138616814,100.7581410362
+3,0.79,1,-0.72122514878988,0.0054555724256439,-88851.000217489,-2.5380194474295,-0.0251893799419,110789.65701139,-0.1572841239466,-0.19365552867233,0,0,161019.71289729,161185.80298299,161027.90621529,161181.51513083,1.0000508839437,1.0000266026297,100.7575197676,0.843,1,-0.49665602377643,0.00968798880176,100.7575197676
```

Comment:
ompressor:rel_corrected_flow went up by a factor of 10, is that expected?

Reply:
And apparently the open one wasn't (and still isn't) running to completion (we're only testing the first few time steps), so I'll need to fix that too...

Reply:
See https://github.com/idaholab/large_media/pull/62